samples/samples-react/src/App.js

```diff
@@ -0,0 +1,62 @@
+import {Workbook} from 'exceljs';
+import React, {useCallback, useState} from 'react';
+import {Importer} from 'xlsx-import/lib/Importer';
+import './App.css';
+import {
+  getInvoiceConfig,
+  Invoice
+} from './components/Invoice';
+
+
+const importInvoice = (importer) => {
+  const config = getInvoiceConfig();
+
+  const {date, dueDate} = importer.getAllItems(config.misc)[0];
+  const seller = importer.getAllItems(config.seller)[0];
+  const buyer = importer.getAllItems(config.buyer)[0];
+  const items = importer.getAllItems(config.items);
+
+  const total = items.reduce((p, c) => p + c.price, 0);
+
+  return {
+    buyer,
+    date,
+    dueDate,
+    items,
+    seller,
+    total
+  };
+};
+
+async function onDownloadInvoice() {
+  return fetch('/invoice.xlsx').then(r => r.blob());
+}
+
+function App() {
+  const [invoice, setInvoice] = useState(null);
+  const onUpload = useCallback(async () => {
+    const invoiceFile = await onDownloadInvoice();
+    const reader = new FileReader();
+    reader.readAsArrayBuffer(invoiceFile);
+    reader.addEventListener('loadend', async (e) => {
+      const wb = new Workbook();
+      await wb.xlsx.load(reader.result);
+      const importer = new Importer(wb);
+      setInvoice(importInvoice(importer));
+    })
+  }, []);
+  return (
+      <div className="App">
```

Comment:
could I propose to add GH icon with link into repo?

Reply:
Where icon should be added?

Reply:
I hadn't specified place. What are you thing about fixed right top?

Reply:
Ah, Ok, I'll think of something